### DISCUSSION TITLE
Changing Fill-Color of Notes

### DISCUSSION THREAD

**Discussion by Soren74** [0xfe/vexflow#1535] [Q&A] [answered]
Is it really possible to change the color of the notes? 
I found the attached image in the "tests" section of the website (https://www.vexflow.com/tests/)
And I also see in the `element.ts` document a property called `fillStyle` -- I just can't figure out how to access it.

So for example if I have:

            new StaveNote({ keys: ["b/4"], duration: "qr" }),

How can I set it's `fillStyle` property to a specific color?

Would really appreciate any help would this.

Thank you!

```
export interface ElementStyle {
... 
fillStyle?: string;
  /**
   * CSS color used with context stroke command.
   *
   * Examples: 'red', '#ff0000', '#ff000010', 'rgb(255,0,0)'
   *
   * See [CSS Legal Color Values](https://www.w3schools.com/cssref/css_colors_legal.asp)
   */
```
![Blue-Notes](https://user-images.githubusercontent.com/57606699/225802495-5fecfea6-30b6-4d1f-94f0-c06762c097e0.png)

**Comment by rvilarl** [comment, marked as answer]
You have to use `setStyle`, example attached

https://jsfiddle.net/Lh81o4wg/

**Reply by Soren74** [reply]
Wow - so simple! 
Thank you so much!

**Reply by Soren74** [reply]
wait - can the canvas itself be painted? 
Meaning the canvas upon which the entire Stave is drawn?

I tried the following - and it's not working: 

         const context = renderer.getContext();
         context.backgroundFillStyle = "red";

**Comment by ronyeh** [comment]
A good way to figure out how to use VexFlow is to treat the `vexflow/tests/` folder as example code. If you dig around in there, you'll start to get a sense of how to work with VexFlow: https://github.com/0xfe/vexflow/tree/master/tests

https://github.com/0xfe/vexflow/blob/9d7ab821bf40595a86f7b769ba212b9482869c2b/tests/stavenote_tests.ts#L664-L682

**Comment by Soren74** [comment]
Thank you for the answer Sir - much appreciated!
